docs/config.md

```diff
@@ -663,15 +663,27 @@ java -Dspring.config.location=file:/path/to/your/myconfig.yaml -jar promregator-
 #### Option "promregator.authenticator.oauth2xsuaa.tokenServiceURL" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA)
 Specifies the URL of the OAuth2 endpoint, which contains the token service of your authorization server in case of global authentication. Typically, this is the endpoint with the path `/oauth/token`, as Promregator will try to perform to establish a ["Client Credentials"-based authentication](https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2#grant-type-client-credentials).
 
+#### Option "promregator.authenticator.oauth2xsuaa.xsuaaServiceURL"
+Like `promregator.authenticator.oauth2xsuaa.tokenServiceURL`, but without trailing `/oauth/token`.
```

Comment:
```suggestion
Like `promregator.authenticator.oauth2xsuaa.tokenServiceURL`, but without trailing `/oauth/token`.

If using `promregator.authenticator.type=OAuth2XSUAA` either `promregator.authenticator.oauth2xsuaa.tokenServiceURL` or `promregator.authenticator.oauth2xsuaa.xsuaaServiceURL` must be provided. If both are provided, `promregator.authenticator.oauth2xsuaa.xsuaaServiceURL` takes precedence.
```

Reply:
Sounds a little as if `tokenServiceURL` was still the "right option". I have a little the impression that `xsuaaServiceURL` is our favored approach in the meantime (see also replacements in examples below). So documentation tends into the "wrong" direction.

Reply:
The parameters `tokenServiceURL` and `xsuaaServiceURL` are very close to each other. So I would like to propose to deprecate `tokenServiceURL`. That means we enforce URLs like `https://example.org:1234` which are transformed internally to `https://example.org:1234/oauth/token`.

Other approach: It would also be possible to keep the `tokenServiceUrl` and to implement a `NullOAuth2ServiceEndpointsProvider` which does not modify the provided url(s). When we follow this approach we would have `tokenServiceUrl` and `tokenServiceCertUrl`. Should I implement it this way? Would be easy to implement.